alchemist/alchemist-loading/src/main/java/it/unibo/alchemist/loader/YamlLoader.java

```diff
@@ -335,6 +339,12 @@ public YamlLoader(final Reader source) {
         } else {
             throw new IllegalAlchemistYAMLException("Exports must be a YAML map.");
         }
+        final Object dependencies = rawContents.get(REMOTE_DEPENDENCIES);
+        if (dependencies == null) {
+            this.dependencies = ImmutableList.of();
+        } else {
+            this.dependencies = ImmutableList.copyOf((List<String>) dependencies);
```

Comment:
This may raise an unclear classcastexception at runtime. Please, leave at least a TODO comment about this fact, I'll get it fixed in future.